apps/frontend/src/pages/auth/auth.tsx

```diff
@@ -13,7 +11,6 @@ import {
 } from "~/modules/users/users.js";
 
 import { SignInForm, SignUpForm } from "./components/components.js";
-import styles from "./style.module.css";
```

Comment:
I think we also need to delete the CSS module file.

Reply:
It is already deleted in this PR